frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -45,7 +56,7 @@ export const TaskOverviewPage = () => {
           <div className={classes(css.buttons)}>
             {siblingTasks.map(({ id, type }) => (
               <IconButton
-                key={id}
+                key={type}
```

Comment:
In addition, the id key caused errors sometimes.

Reply:
I'd guess `undefined` key could cause some errors